regex-posix.cabal

```diff
@@ -1,14 +1,15 @@
 cabal-version:          1.12
 name:                   regex-posix
 version:                0.96.0.0
+x-revision:             2
 
 build-type:             Simple
 license:                BSD3
 license-file:           LICENSE
 copyright:              Copyright (c) 2007-2010, Christopher Kuklewicz
 author:                 Christopher Kuklewicz
 maintainer:             hvr@gnu.org
```

Comment:
could you bump the maintainer list here?

Reply:
Who should be on the list?

Reply:
Clarified that with @hvr.